core/mainstage.go

```diff
@@ -356,18 +366,28 @@ func (st *Stage) newRenderWindow() *renderWindow {
 		Icon:      appIconImages(),
 		Size:      st.Scene.SceneGeom.Size,
 		StdPixels: false,
+		Screen:    st.Screen,
+	}
+	if !st.Resizable {
```

Comment:
Does this fix #1329? If so, we should mention it and also add support for the same flag in dialog.